contents/docs/feature-flags/installation.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Feature flag installation
+sidebar: Docs
+showTitle: true
+availability:
+    free: none
+    selfServe: full
+    enterprise: full
+---
+
+Feature flags enable safer deployment and roll back of new features and code. This helps roll out new features to your users in a managed way. If something goes wrong, flags enable roll backs without having to re-deploy your application.
+
+Feature flags also help you control access to certain parts of your product, such as only showing paid features to users with an active subscription or [early access features[(/docs/feature-flags/early-access-feature-management) to users enrolled in a beta. 
+
+To start using feature flags, install PostHog:
```

Comment:
More of an observation, but we really need to come up with a better solution for all the libraries. This horizontal scrolling thing is ugly and makes these pages feel clunky. Doesn't create joy! 

<img width="711" alt="Screenshot 2023-08-23 at 12 08 50" src="https://github.com/PostHog/posthog.com/assets/92976667/4daac781-44d2-4827-8d2f-79b4f5d950d1">

Reply:
Agree! Will chat to Cory when he's back